index.html

```diff
@@ -416,65 +416,172 @@ <h2 id="liaisons">
-      repository for the project that they are interested in. This may be in
-      the form of a pull request (preferred), by raising an issue, or by adding
-      a comment to an existing issue.
+    Substantive Contributions to Specifications can only be made by Community 
+    Group Participants who have agreed to the 
+    <a href="http://www.w3.org/community/about/agreements/cla/">W3C Community
+      Contributor License Agreement (CLA)</a>. 
     </p>
+    
     <p>
-      Specifications created for proposals in the Community Group must use the
+      Specifications created in the Community Group must use the
       <a href=
       "http://www.w3.org/Consortium/Legal/2015/copyright-software-and-document">
-      W3C Software and Document License</a>.
+      W3C Software and Document License</a>.  All other documents produced 
+      by the group should use that License where possible. 
     </p>
+
+    <p class="remove">
+      {TBD: if CG doesn't use GitHub replace the remaining paragraphs in this 
+      section with: "All Contributions are made on the groups public mail list
+      or public contrib list"} 
+    </p>
+    
+    <p>
+      Community Group participants agree to make all contributions 
+      in the GitHub repo the group is using for the particular document. 
+      This may be in the form of a pull request (preferred), by raising an issue, 
+      or by adding a comment to an existing issue.
+    </p>
+
     <p>
       All Github repositories attached to the Community Group must contain a
-      copy of the <a href="CONTRIBUTING.md">CONTRIBUTING</a> and <a href=
-      "LICENSE.md">LICENSE</a> files.
+      copy of the 
+      <a href="https://github.com/w3c/licenses/blob/master/CG-CONTRIBUTING.md">CONTRIBUTING</a>
+      and <a href=
+      "https://github.com/w3c/licenses/blob/master/CG-LICENSE.md">LICENSE</a> files.
+    </p>
+   <p class="note">Proposed charter deleted last section of this paragraph</p>
+    <p>
+      Note: this Community Group will not use a contrib mailing list for contributions 
+      since all contributions will be tracked via Github mechanisms 
+      (e.g. pull requests). For the same reason, the Community Group 
+      mailing list must not be used for making or discussing substantive contributions
+      to Specifications.
     </p>
+    
+    <h2 id="transparency">
+      Transparency
+    </h2>
+    
     <p>
-      Note: this Community Group will not use a <em>contrib</em> mailing list
-      for contributions since all contributions will be tracked via Github
-      mechanisms (e.g. pull requests).
+      The group will conduct all of its technical work in public. If the group 
+      uses GitHub, all technical work will occur in its GitHub repositories 
+      (and not in mailing list discussions). This is to ensure contributions 
+      can be tracked through a software tool.
     </p>
+  
+   <p class="note">Proposed charter deleted this paragraph</p>  
```

Comment:
yes, this part can go and it will match the current template

>   Note: this Community Group will not use a <em>contrib</em> mailing list  for contributions since all contributions will be tracked via Github  mechanisms (e.g. pull requests).  The group will conduct all of its technical work in public. If the group  uses GitHub, all technical work will occur in its GitHub repositories  (and not in mailing list discussions). This is to ensure contributions   can be tracked through a software tool.